src/server/vnparse.c

```diff
@@ -2931,11 +3262,102 @@ add_to_vnl(vnl_t **vnlp, char *noden, char *keyw, char *keyval)
 		free(msgbuf);
 		return (1);
 	}
-
+#endif
 	return (0);
 }
 
+#if !(defined(PBS_MOM) || defined(PBS_PYTHON))
 /** 
+ * @brief
+ *	Check if other resources in 'have' satisfy the remaining
+ *	 resources from 'need'.
+ *
+ * @param[in]	need - the need value of resc_limit_t type.
+ * @param[in]	have - the have value of resc_limit_t type.
+ *
+ * @return int
+ *	1 - if not satisfied
+ *	0 - otherwise
+ * -1 - error
```

Comment:
Indentation off.

Reply:
corrected in commit https://github.com/PBSPro/pbspro/pull/1452/commits/8f309272ad00f7587c830cad07238cb40598e24a